sdk/test-utils/test-utils/src/tracing/chaiAzureTrace.ts

```diff
@@ -128,25 +99,3 @@ function getSpanGraph(traceId: string, instrumenter: MockInstrumenter): SpanGrap
     roots,
   };
 }
-
-/* eslint-disable @typescript-eslint/no-namespace */
-declare global {
-  export namespace Chai {
-    interface Assertion {
-      supportTracing<T>(expectedSpanNames: string[], options?: T): Promise<void>;
-    }
-    interface Assert {
-      supportsTracing<
-        Options extends { tracingOptions?: OperationTracingOptions },
-        Callback extends (options: Options) => Promise<unknown>
-      >(
-        callback: Callback,
-        expectedSpanNames: string[],
-        options?: Options,
-        thisArg?: ThisParameterType<Callback>
-      ): Promise<void>;
-    }
-  }
-}
-
-export { chaiAzureTrace };
```

Comment:
I see that we used to have this, but I wonder if this is actually useful or just seems convenient. Without this, it might look much simpler, perhaps specially for new hires?

Reply:
Yeah, totally fair. I'll definitely bring this up on our next team meeting and see if we can get a consensus